src/com/sahokia/kos/concerttickets/annotations/NullableWarning.java

```diff
@@ -0,0 +1,11 @@
+package com.sahokia.kos.concerttickets.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface NullableWarning {
```

Comment:
could also add some field for the annotation to use it later as a param :)

Reply:
Add a "message" field to use later as a param